docs/proposals/lifelong-learning/unstructured-lifelong-learning.md

```diff
@@ -0,0 +1,247 @@
+* [Unstructured Lifelong Learning](#lifelong-learning)
+   * [Motivation](#motivation)
+     * [Goals](#goals)
+     * [Non\-goals](#non-goals)
+   * [Proposal](#proposal)
+     * [Use Cases](#use-cases)
+   * [Design Details](#design-details)
+     * [CRD API Group and Version](#crd-api-group-and-version)
+     * [Lifelong learning CRD](#lifelong-learning-crd)
+     * [Lifelong learning type definition](#lifelong-learning-job-type-definition)
+     * [Lifelong learning sample](#lifelong-learning-job-sample)
+     * [Validation](#validation)
+   * [Controller Design](#controller-design)
+     * [Lifelong Learning Controller](#lifelong-learning-controller)
+     * [Downstream Controller](#downstream-controller)
+     * [Upstream Controller](#upstream-controller)
+     * [Details of api between GM(cloud) and LC(edge)](#details-of-api-between-gmcloud-and-lcedge)
+   * [Workers Communication](#workers-communication)
+
+# Unstructured Lifelong Learning
+## Motivation
+
+Edge cloud synergy lifelong learning has been proposed and realized through Sedna in [ATCII example](https://github.com/kubeedge/sedna/blob/main/examples/lifelong_learning/atcii/README.md) which handles structured data. However, the last version of Sedna lifelong learning scheme has the following limitations.
```

Comment:
Might specify which the mentioned version is.

Reply:
The version has been specified.